app/src/openxr/cpp/DeviceDelegateOpenXR.cpp

```diff
@@ -1192,6 +1192,7 @@ DeviceDelegateOpenXR::EndFrame(const FrameEndMode aEndMode) {
 
   // Add VR video layer
   if (m.equirectLayer && m.equirectLayer->IsDrawRequested()) {
+    m.equirectLayer->SetReorientPose(MatrixToXrPose(GetReorientTransform()));
```

Comment:
We don't really need a new method. Let's use `Update()`

Reply:
I thought about that. The problem is that it would mean changing the signature of `OpenXRLayerEquirect::Update`, which currently overrides a method from the superclass `OpenXRLayer`. Is that what you are suggesting?

Reply:
Right. If I am not wrong the base layer does get the aPose attribute but it does not do anything with it. So it's a nice opportunity to give this support to all types of layers.